tools/delay_detector/src/lib.rs

```diff
@@ -1,18 +1,24 @@
+#[cfg(features = "delay_detector")]
```

Comment:
```suggestion
#[cfg(feature = "delay_detector")]
```

every cfg in this file has this typo.